### PR TITLE
Fix cad graphical glitches

### PR DESCRIPTION
**Pull request by olivierdalang**
Hi @m-kuhn ,

Opening a PR here again in case you want to review prior to actual PR

before :
before

after :
after

Was this what you were talking about ? Let me know if you want me to open the PR on the main repo.

### CODE REVIEW

File: src/gui/qgsadvanceddigitizingdockwidget.h [hunk 1]
```diff
@@ -281,11 +281,17 @@ class GUI_EXPORT QgsAdvancedDigitizingDockWidget : public QgsDockWidget, private
     void clearPoints();
 
     /**
-     * Adds point to the CAD point list
-     * \since QGIS 3.0
-     */
+    * Adds point to the CAD point list
```
Comment (m-kuhn) [inline src/gui/qgsadvanceddigitizingdockwidget.h]: The old styling is what's normally used.

**Comment by m-kuhn** [comment]
Thanks!
Looks good to me, just the docstrings have a slightly misaligned indentation.

**Comment by olivierdalang** [comment]
Oops thought that would be fixed by astyle, I'll fix and open the PR. Should I put the same mention as last time in the commit ? (`"Sponsored by Kanton Schaffhausen in collaboration with OPENGIS.ch"`)

**Comment by m-kuhn** [comment]
Opening the PR sounds good.
I don't think this note is needed, it's mostly interesting for new features which will be listed in the visual changelog where we and the customer will have some visibility (next to you who will be listed as author anyway).

**Comment by olivierdalang** [comment]
PR opened on the main repo

**Pull request closed without merging**